atoma-event-subscribe/sui/src/main.rs

```diff
@@ -20,36 +20,36 @@ struct Args {
 
 #[tokio::main]
 async fn main() -> Result<(), SuiSubscriberError> {
-    tracing_subscriber::fmt::init();
-
-    let args = Args::parse();
-    let package_id = ObjectID::from_hex_literal(&args.package_id)?;
-    let http_url = args.http_addr;
-    let ws_url = args.ws_addr;
-
-    let (event_sender, mut event_receiver) = tokio::sync::mpsc::channel(32);
-
-    let event_subscriber = SuiSubscriber::new(
-        1,
-        &http_url,
-        Some(&ws_url),
-        package_id,
-        event_sender,
-        Some(Duration::from_secs(5 * 60)),
-    )
-    .await?;
-
-    tokio::spawn(async move {
-        info!("initializing subscribe");
-        if let Err(err) = event_subscriber.subscribe().await {
-            error!("Failed to subscribe: {:?}", err);
-        }
-        Ok::<_, SuiSubscriberError>(())
-    });
-
-    while let Some(event) = event_receiver.recv().await {
-        info!("Processed a new event: {:?}", event);
-    }
+    //     tracing_subscriber::fmt::init();
```

Comment:
Why ?

Reply:
because now the event subscriber expects channel from input manager. Why do we even have standalone binaries for this? We will never run just event subscriber.

Reply:
It is helpful to have internal testing, but we can remove it